lizard_auth_server/views_api_v2.py

```diff
@@ -466,51 +466,54 @@ def form_valid(self, form):
         # Try to find the user first. You can have multiple matches.
         matching_users = User.objects.filter(
             email__iexact=form.cleaned_data['email'])
-        user = None
-        status_code = 200
+
         if matching_users:
+
+            # Return statuscode 409 (conflict) when email address is
+            # already in use.
             if len(matching_users) > 1:
                 logger.debug(
                     "More than one user found for '%s', returning the first",
                     form.cleaned_data['email'])
             user = matching_users[0]
-            logger.info("Found existing user %s, giving that one to %s",
+            logger.info("Found existing user based on email %s in portal: %s",
```

Comment:
Een user zit niet echt in een portal. Ik zou "in portal %s" gewoon weghalen, dan is het ook goed.